doc/constitution.md

```diff
@@ -99,7 +99,7 @@ Substantial amendments to the Nix Community Values require 90% agreement in a po
 
 #### Ordinary decisions
 
-Everything within the authority of SC that doesn't require an extraordinary decision can be decided with a simple majority (more than 50%) of a full SC (vacant seats count as abstaining).
+Everything within the authority of SC that doesn't require an extraordinary decision can be decided with a simple majority (more than 50%) of a full SC (vacant seats count as abstaining, except when voting for reelection).
```

Comment:
The current phrasing creates an ambiguity regarding reelection votes. 

It explicitly states that vacant seats do not count as `abstaining` but it fails to specify how they should be treated. Making explicit that they're not counted for or against, but are in fact not counted at all may be a sensible clarification.

```suggestion
Everything within the authority of SC that doesn't require an extraordinary decision can be decided with a simple majority (more than 50%) of a full SC (vacant seats count as abstaining, except when voting for reelection, in which case they are excluded from the total).
```